bindings/python/iota_sdk/client/_high_level_api.py

```diff
@@ -44,29 +46,30 @@ def as_dict(self):
 
 class HighLevelAPI():
 
-    def get_outputs(self, output_ids: List[OutputId]):
+    def get_outputs(self, output_ids: List[OutputId]) -> List[OutputWithMetadata]:
         """Fetch OutputResponse from provided OutputIds (requests are sent in parallel).
```

Comment:
OutputResponse is mentioned here and below, even though i dont think they exist in python. should rename it to idk.. "fetch OutputWithMetadata"

Reply:
Done